Reflections/Week_2_Reflection.md

```diff
@@ -0,0 +1,14 @@
+# Weekly Reflection- Week Two
+
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+  * I have created a new character and began work on their dialogue. I have added new optional objectives for the quest. I have created the reputation system needed to track the player's progress with certain optional objectives.
+  
+2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+  * I have had issues finding time to work. I have started to manage my homework better, so this should become less of a problem. I still need to finish up creating all the new objectives and dialogue functions, such as returning items to Mario's boss to satisfy optional objectives. I have learned how to properly add reputation values to the player.
+
+3. What do you plan to accomplish in the following week? (Please list at least 3 concrete "S.M.A.R.T." goals, along with estimated number of hours to complete, or day to complete)
+  * I will finish up the optional objectives of the quest. This should take around three hours. I will add those objectives to the quest properly. This should take around three hours, including testing time. I will begin to create new areas for the quest to take place in. This will be the most time expensive task, and I am avoiding it until everything else is done. I assume this will take around four hours for each area.
```

Comment:
Great goals. I would like to encourage that you may tackle the areas sooner rather than later-- games are simulations after all, and simulating an environment is no easy task, as you elude to here.